src/Entity/Dinosaur.php

```diff
@@ -3,13 +3,28 @@
 namespace App\Entity;
 
 use App\Enum\HealthStatus;
+use App\Repository\DinosaurRepository;
+use Doctrine\ORM\Mapping as ORM;
 
+#[ORM\Entity(repositoryClass: DinosaurRepository::class)]
 class Dinosaur
 {
+    #[ORM\Id, ORM\GeneratedValue, ORM\Column]
+    private int $id;
+
+    #[ORM\Column]
     private string $name;
+
+    #[ORM\Column]
     private string $genus;
+
+    #[ORM\Column]
     private int $length;
+
+    #[ORM\Column]
     private string $enclosure;
+
+    #[ORM\Column]
```

Comment:
The entity is a bit non-traditional when it comes to nullability of properties - e.g. `make:entity` would do `?string $enclosure = null` to all of these & also the getters would return a nullable type. Unless you've done this on purpose, let's make this look "traditional"

Reply:
These values are never `null` because we set them in the existing constructor w/ default values. 

If we remove the constructor args / defaults, we'll also have to refactor some of the existing tests / code to use setters instead. While there wouldnt be a whole lot to change - think it may throw off someone who goes for ep1 to ep2 and like, huh - where'd this go... especially if they arn't downloading the new start code. But i'm down for whatever.

Reply:
Ah, I forgot about the constructor. All good then as it is